src/components/GuideLegendaries/index.js

```diff
@@ -1,201 +1,447 @@
+import { Link } from 'react-router-dom'
+import { TIER_COLORS, MAX_TIERS } from '../../constants/list'
+import getInitialListData from '../../helpers/getInitialListData'
 import React from 'react'
 import Info from '../Info'
 import Guide from '../Guide'
+import ListBuilderTier from '../ListBuilderTier'
 import Title from '../Title'
 import CardLink from '../CardLink'
 import getGuide from '../../helpers/getGuide'
 
 const guide = getGuide('LEGENDARIES_GUIDE')
+const TIER_LIST =
+  'S - Always Usable,N8N59N69N46S19F21;A - Usable in Most%2FSome Decks,N58I2I17I22W8W23F12;B - Usable in Very Few Decks,N35N48S21W10F23;C - Never Use,N77N76S3I29/display'
 
 export default React.memo(function GuideLegendaries(props) {
+  const tiers = getInitialListData(TIER_LIST)
+
   return (
     <Guide {...guide}>
       <p>
-        Legendaries are shiny, very rare and may often look good at first sight.
-        Most new players tend to automatically include their legendary cards in
-        their deck. Sadly, most of them have better, similar, less rare
-        substitutes at low levels.
+        Legendaries are shiny, very rare, and may often look good at first
+        sight. Most new players tend to automatically include their new
+        legendary cards in their deck. Sadly, some of them have better, similar,
+        or less rare substitutes at low levels. But some will definitely strike
+        fear in the battlefield.
       </p>
 
-      <p>
-        This guide aims to give a definite answer to the question:{' '}
-        <span className='Highlight'>should I use this level 1 legendary?</span>{' '}
-        We’ll also see when some legendary cards can be used for maximum
-        potential.
+      <p>This guide aims to give a definite answer to the question:</p>
+
+      <hr />
+      <p
+        style={{ fontSize: '135%', textAlign: 'center', color: 'var(--beige)' }}
+      >
+        Should I use this level 1 legendary?
       </p>
+      <hr />
 
-      <Info icon='compass' title='Outdated guide'>
+      <Info icon='info' title='About Equals Meta'>
         <p>
-          This guide has not been updated since its first release and might be
-          slightly outdated. Tread with caution.
+          This guide focuses on the scope of Ranked at the very early levels
+          (1-3). This guide is <span className='Highlight'>not</span> an
+          accurate representation of the{' '}
+          <span className='Highlight'>Equals</span> meta. For more information,
+          please visit the{' '}
+          <span className='Highlight'>
```

Comment:
Let’s remove this highlight please (and the one for the next link). Links should look consistent throughout the site. :)